pages/docs/index.md

```diff
@@ -3,3 +3,31 @@ title: Overview
 ---
 
 # {% $markdoc.frontmatter.title %}
+
+## Example Next.js tags
```

Comment:
We already have a page for this https://markdoc.dev/docs/nextjs

Reply:
Yes but the point of this PR is to add example code to `markdoc-starter`.